src/Relogify/Settings.fs

```diff
@@ -263,7 +256,7 @@ let dialogBody (model: Model) dispatch =
 
                              yield View.Button(
                                      text = "Cancel",
-                                     isVisible = canCancelDialog model,
+                                     isVisible = isNotEmpty savedCommunityName,
```

Comment:
Should we also add `commandCanExecute` here? Like we did in some other places.

Reply:
I think it's better if the button is not visible at all - this would make most sense during the first run.